cogniac/edgeflow.py

```diff
@@ -350,3 +350,38 @@ def get_next(url):
                 if limit and count == limit:
                     return
             url = resp['paging'].get('next')
+
+
+    def get_xput_stats(self, last_x_minutes=5):
```

Comment:
probably should use start_time and end_time here to make this more closely match the API.  can default the values to last five minutes if not specified by user

Reply:
We can do that. But does someone actually provide time in the timestamp format? It's not very intuitive.